Move AKSScalarTerraformDeploymentGuide.md from scalar-kubernetes (#279)
scalar-labs/scalar-terraform: docs/CloudPrivileges.md

}
```

> ### A Role for Deployment with scalar-kubernetes

Reviewer: ```suggestion
### A Role for Deployment with scalar-terraform kubernetes module
```
Correct?

Author: Yes, Fixed in 3ba5236

To interact with Azure APIs, an AKS cluster requires an Azure Active Directory (AD) service principal, which is used to create and manage other Azure resources such as an Azure load balancer.

> In scalar-kubernetes, the terraform module is the one who creates a service principal and assigns permissions to it and deletes it, so we recommend assigning `User Access Administrator` to operators of Terraform.

Reviewer: ```suggestion
When creating a Kubernetes cluster with the `kubernetes` module, the module is the one that creates a service principal and assigns permissions to it and deletes it, so we recommend assigning `User Access Administrator` to operators of Terraform.
```
correct?

Author: Fixed in 3ba5236.
